### ISSUE TITLE
Add Versions To Requirements Explicitly

### ISSUE THREAD

**Issue by becurrie** [becurrie/titandash#248]
Some more investigation needed maybe... Seems like some users are getting a missing sphinx module when running the `titandash.bat` file after running the initial `setup.bat`... I'm unsure why some people get it and some people don't.

We should be able to explicitly add the module to the requirements file to avoid this issue.

```
PS C:\Users\Blaine\Desktop\titandash-1.6.2.1> workon titandash "&" cd titanbot "&" python titandash.py
Traceback (most recent call last):
  File "titandash.py", line 19, in <module>
    import PySimpleGUIWx as sgw
  File "C:\Users\Blaine\Envs\titandash\lib\site-packages\PySimpleGUIWx__init__.py", line 2, in <module>
    from .PySimpleGUIWx import *
  File "C:\Users\Blaine\Envs\titandash\lib\site-packages\PySimpleGUIWx\PySimpleGUIWx.py", line 2, in <module>
    from sphinx.ext.autosummary import autosummary_table
ModuleNotFoundError: No module named 'sphinx'
```

We do not need sphinx explicitly, but rather, we need to explicitly define the requirements versions of modules being used... A pip freeze will work for this to find out which version is being used (in my working env). And adding those to the file.

**Comment by becurrie** [comment]
Resolved by #252

**Comment by becurrie** [comment]
Resolved by #252
